### ISSUE TITLE
Server console not refreshing automatically upon restart (and disk/ram/cpu values not showing)

### ISSUE THREAD

**Issue by JanDevDE** [pterodactyl/panel#2298]
**Background (please complete the following information):**
* Panel or Daemon: Both
* Version of Panel/Daemon: rc2
* Server's OS:  Ubuntu 20.04
* Your Computer's OS & Browser: Chrome

**Describe the bug**
Sometimes (yea, sorry), when restarting a server the console stops auto-refreshing (sending commands over WebSockets still work) and the disk/ram/CPU values are not being shown anymore (0 % values) on the server page, however the server overview page shows the values just fine.

Refreshing the server page refreshes the console, but as described the statistics stay at 0%. Only a daemon restart fixes the auto-refresh and the statistics.

![image](https://user-images.githubusercontent.com/11154004/91771259-fabc4480-ebe2-11ea-9363-9a44a2f8b008.png)

![image](https://user-images.githubusercontent.com/11154004/91771288-06a80680-ebe3-11ea-9d3d-86f73bd4a99b.png)

**To Reproduce**
1. Restart a server until the problem described above appears
2. Restart wings to fix the problem (Or just refresh the page to see the newest commands run)

**Comment by JanDevDE** [comment]
No errors in console, in wings or in panel log.

**Comment by Sir3lit** [comment]
This also happens randomly to a server that hasn't been restated in 8 plus hours.

**Comment by GiovaniFerraroTrivelli** [comment]
I am facing the same issue. 

-     Version of Panel/Daemon: rc3
-     Server's OS: Debian 9
-     Your Computer's OS & Browser: Firefox/Chrome

**Comment by DaneEveritt** [comment]
I'm struggling to reproduce this at all, especially if the only information I can get is "let it run for awhile" or something. I sat here restarting a server for 20 minutes and nothing broke. I don't doubt that there is an issue, but with the information given I cannot track down what could be the issue.

**Comment by DaneEveritt** [comment]
@Sir3lit @JanDevDE Please try running this build if you don't mind: https://github.com/pterodactyl/wings/actions/runs/238861418

It won't fix your issue but it should add some additional logging around the code that handles polling for resources and console output. This should help me at least figure out if it is something with wings or not.

Please also make sure it is running in debug mode, otherwise context gets lost.

**Comment by Sir3lit** [comment]
The correct location for the file is https://github.com/pterodactyl/wings/actions/runs/238861416

**Comment by TheKosciany** [comment]
Hello,

I came here, because I have the same issue. The problem is exactly the same as described by @JanDevDE 
I run the server, everything shows fine, after the couple of hours of running the server is not showing the statistics and only refreshing the whole page updates the console. Only stopping and starting the wings again fixes it.

What can I do to provide you the logs necessary to debug the issue?

**Comment by JanDevDE** [comment]
@TheKosciany Install the daemon provided mentioned in the comment before yours, then change debug to true in wings config.

**Comment by elliotnichols** [comment]
Dane if you would like to contact me on discord Elliot#9128 I am more than happy to sit with you in a call on my instance which has a server that has this issue 100% of the time.

**Comment by elliotnichols** [comment]
Right, I am lead to believe that this issue is caused by the number of files present within the server. 
I noticed this when I installed a plugin that is known to generate a lot of files then caused the console to break similar to one of our other servers. 

Upon deleting the files the plugin created the console begins to work again.
Upon generating the files again via installing the plugin the console begins to have the refresh bug again.

This works for any number of files. It is lso fixable by deleting an excessive number of log files provided it drops below the arbitrary limit that I do not know the specifics of. You should bear in mind that it could be caused by the number of files or maybe the file size of the container?  

I have triple checked and this is definitely the cause, at least on my instance.

**Comment by JanDevDE** [comment]
> Right, I am lead to believe that this issue is caused by the number of files present within the server.
> I noticed this when I installed a plugin that is known to generate a lot of files then caused the console to break similar to one of our other servers.
> 
> Upon deleting the files the plugin created the console begins to work again.
> Upon generating the files again via installing the plugin the console begins to have the refresh bug again.
> 
> This works for any number of files. It is lso fixable by deleting an excessive number of log files provided it drops below the arbitrary limit that I do not know the specifics of. You should bear in mind that it could be caused by the number of files or maybe the file size of the container?
> 
> I have triple checked and this is definitely the cause, at least on my instance.

I cannot confirm that. Issue also appears without any file changes on my end.

**Comment by vlakyan** [comment]
There is a problem.
Console only after page refresh.
The rest of the data is not shown except for the HDD
![image](https://user-images.githubusercontent.com/53228824/92367770-4b1a3180-f110-11ea-97f8-7372ecab031c.png)
![image](https://user-images.githubusercontent.com/53228824/92367837-6127f200-f110-11ea-809c-51abc1738168.png)
Servers are running, docker sees them, sftp is working properly.

**Comment by JanDevDE** [comment]
@vlakyan I don't think this is adding any value to the issue.

**Comment by vlakyan** [comment]
> @vlakyan I don't think this is adding any value to the issue.

The problem appeared after switching from daemon to wings

**Comment by JanDevDE** [comment]
@vlakyan Obviously.

**Comment by MrPvTDagger** [comment]
I'm also facing the same issue:

Version of Panel/Daemon: rc3
Server's OS: Ubuntu 18 LTS
Your Computer's OS & Browser: Opera /Firefox

**Comment by TheNickSkater** [comment]
Similar issue here.
Though for me it only happens for newly created Servers.
After the installation has finished, I press start and it only shows the message for it calculating disk usage and boom, it does not update after that unless I restart wings. It does not affect other servers.
I have no errors in the log.

**Comment by ThatOneMineman** [comment]
it happens when i refresh page when the server is starting

**Comment by gOOvER** [comment]
For me it happend, when i often start and stop server. Then i need to restarts wings

**Comment by Sir3lit** [comment]
Welcome to RC4 where the world is on fire...

The issue got worse. If the server is started with RC4. you now get nothing unless you refresh the page... every...time...

When you click the Start button you are greeted with...
```
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: Checking server disk space usage, this could take a few seconds...
```
In the container and that's about it... unless you refresh the page.

Wings log when starting the server...
```
DEBUG: [Sep 11 06:35:21.741] GET /api/servers/ae7ba8bb-53de-4b26-980d-b34ecaf9821f/ws client_ip=69.69.69.69 latency=179.941848ms status=200
 INFO: [Sep 11 06:35:24.896] syncing server configuration with panel server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f
DEBUG: [Sep 11 06:35:24.896] making request to external HTTP endpoint endpoint=https://test.justmyrandomstuff.com/api/remote/servers/ae7ba8bb-53de-4b26-980d-b34ecaf9821f headers=map[Accept:[application/vnd.pterodactyl.v1+json] Authorization:[Bearer 7WUITwcB(redacted)] Content-Type:[application/json]] method=GET
DEBUG: [Sep 11 06:35:24.916] syncing stop configuration with configured docker environment server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f
DEBUG: [Sep 11 06:35:24.916] syncing server settings with environment server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f
 INFO: [Sep 11 06:35:24.916] performing server limit modification on-the-fly server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f
DEBUG: [Sep 11 06:35:24.949] saw server status change event server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f status=starting
DEBUG: [Sep 11 06:35:26.142] pulling docker image... this could take a bit of time image=areyouscared/pterodactyl:glibc
DEBUG: [Sep 11 06:35:27.049] starting resource polling for container container_id=ae7ba8bb-53de-4b26-980d-b34ecaf9821f
DEBUG: [Sep 11 06:35:39.010] detected server in running state based on console line output against="  11.960 Hosting game at IP ADDR:({0.0.0.0:25550})" match=Hosting game at server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f
DEBUG: [Sep 11 06:35:39.011] saw server status change event server=ae7ba8bb-53de-4b26-980d-b34ecaf9821f status=running
```

Note that restarting wings does nothing to help like it did with RC3.

**Comment by gOOvER** [comment]
@Sir3lit i can't confirm, that it's on every Server with RC4.

Looks like it works for me for the most servers

**Comment by schrej** [comment]
One user on Discord reported that only restarting the node fixes the issue for him, just restarting wings doesn't.

**Comment by Sir3lit** [comment]
Ways to break console...

Stop server....
Refresh Page...
Instantly start-server once the start button appears.
Observe the broken server.

**Comment by Axhrefx** [comment]
did u opened cloudlflare proxied ?
